app/javascript/components/supplemental_application/SupplementalApplicationPage.js

```diff
@@ -1,48 +1,84 @@
 import React from 'react'
+import { cloneDeep } from 'lodash'
 
-import SupplementalApplicationContainer from './SupplementalApplicationContainer'
 import appPaths from '~/utils/appPaths'
 import mapProps from '~/utils/mapProps'
-import CardLayout from '../layouts/CardLayout'
 import { mapApplication, mapFieldUpdateComment } from '~/components/mappers/soqlToDomain'
 import { updateApplicationAction } from './actions'
 import { mapList } from '~/components/mappers/utils'
+import CardLayout from '../layouts/CardLayout'
+import SupplementalApplicationContainer from './SupplementalApplicationContainer'
 
-const SupplementalApplicationPage = ({ application, statusHistory, formFields, onSubmit, fileBaseUrl }) => {
-  const pageHeader = {
-    title: `${application.name}: ${application.applicant.name}`,
-    breadcrumbs: [
-      { title: 'Lease Ups', link: '/lease-ups' },
-      { title: application.listing.name, link: appPaths.toListingLeaseUps(application.listing.id) },
-      { title: application.name, link: '#' }
-    ]
+class SupplementalApplicationPage extends React.Component {
+
+  constructor(props) {
+    super(props)
+    this.state = {
+      persistedApplication: cloneDeep(props.application),
```

Comment:
not immediately clear to me what "persistedApplication" is.

Reply:
we need to keep and unmodified copy of the persisted application in saleforce, so we can update the application (bottom save) and "individual preference" (panels) independently.

The root of the problem is that both use the same endpoint that update the application + preferences. Each individual save cannot overwrite the previous saved date, so that is why I keep I copy of the persisted data, if someone saves something we update this so the other calls can use it. 

We kind of try to solve this with the less time consuming approach. Another API end point for saving the preferences and application independently might take a bit more time to develop, since it will also involves vertiba. 

confusing? let me know on slack or we can talk on stand up